feature/profile/src/main/java/com/iteneum/profile/presentation/ProfileView.kt

```diff
@@ -28,18 +29,30 @@ import androidx.compose.ui.res.painterResource
 import androidx.compose.ui.res.stringResource
 import androidx.compose.ui.text.font.FontWeight
 import androidx.compose.ui.unit.sp
+import androidx.hilt.navigation.compose.hiltViewModel
 import com.iteneum.designsystem.components.LpFloatingActionIconButton
 import com.iteneum.designsystem.components.LpOutlinedButton
 import com.iteneum.designsystem.theme.LeasePertTheme
 import com.iteneum.designsystem.R as R_DS
 import com.iteneum.profile.R
+import com.iteneum.profile.data.ProfileViewModel
 
 @Composable
 fun ProfileView(
-    /*TODO expected parameters as navigation and view model*/
+    backToDashBoard: () -> Unit /* TODO - ProfileView - use when back arrow component is implemented */ ,
+    profileViewModel: ProfileViewModel = hiltViewModel()
 ) {
-    val sizes = LeasePertTheme.sizes
+    LaunchedEffect(true) {
+        profileViewModel.getInformation()
+    }
+    ProfileContainer()
+}
 
+@Composable
+fun ProfileContainer(
+    profileViewModel: ProfileViewModel = hiltViewModel()
```

Comment:
we don't need to give the viewmodel here, you need to implement the hight order functions for events

Reply:
Got it, done!